lfucg-exactions/server/plats/views.py

```diff
@@ -1,3 +1,135 @@
 from django.shortcuts import render
+import csv
+from django.views.generic import View
+from django.http import HttpResponse
 
-# Create your views here.
+from .models import *
+from .serializers import *
+from accounts.models import *
+from .utils import calculate_lot_balance
+
+class PlatCSVExportView(View):
+     def get(self, request, *args, **kwargs):
+        headers = ['Subdivision',
+            'Developer Account',
+            'Total Acreage',
+            'Buildable Lots',
+            'Non-Buildable Lots',
+            'Plat Type',
+            'Expansion Area',
+            'Unit',
+            'Block',
+            'Section',
+            'Cabinet',
+            'Slide Number',
+            'Non-Sewer Exactions',
+            'Sewer Exactions',
+        ]
+
+        plat = request.GET.get('plat')
+
+        response = HttpResponse(content_type='text')
+        response['Content-Disposition'] = 'attachment; filename="plat_report.csv"'
```

Comment:
could we add a plat_id or name to this filename? (plat_report_shady_acres.csv or plat_report_123.csv)  This way if they generate more than one at once, they will have sensible names.